src/webgpu/api/operation/rendering/depth_clip_clamp.spec.ts

```diff
@@ -41,14 +37,14 @@ have unexpected values then get drawn to the color buffer, which is later checke
     u //
       .combine('format', kDepthStencilFormats)
       .filter(p => kTextureFormatInfo[p.format].depth)
-      .combine('clampDepth', [false, true])
+      .combine('unclippedDepth', [false, true])
```

Comment:
Because we had a bug here, let's add an option here as a regression test. (more extensive testing should happen when we do validation tests for optional features)
```suggestion
      .combine('unclippedDepth', [undefined, false, true])
```